style.css

```diff
@@ -58,14 +59,16 @@ nav .menu-bar {
   left: 50%;
   transform: translateX(-50%);
   width: 375px;
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  height: 93%;
+  background: #f7f7f9;
+  display: grid;
+  place-items: center;
+  gap: 48px;
 }
```

Comment:
- [ ] I have noticed that you have used `grid` on the universal selector :ok_person: . The [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#project--requirements) guide us to use `CSS grid` in order to layout the cards.